website/content/en/preview/tasks/amitasks.md

```diff
@@ -13,26 +13,26 @@ Later, there are tasks that describe the ways that you can intervene to assert c
 Features for managing AMIs described here should be considered as part of the larger upgrade policies that you have for your clusters.
 See [How do I upgrade an EKS Cluster with Karpenter]({{< relref "../faq/#how-do-i-upgrade-an-eks-cluster-with-karpenter" >}}) for details on this process. 
 
-## How Karpenter assigns AMIs to nodes by default
+## How Karpenter assigns AMIs to nodes
 
-If you do nothing to modify how Karpenter handles AMIs, here is how Karpenter assigns AMIs nodes:
+Here is how Karpenter assigns AMIs nodes:
 
 * When you create an `EC2NodeClass`, you are required to set the family of AMIs to use. For example, for the AL2 family, you would set `amiFamily: AL2`.
-* With that `amiFamily` set, any time Karpenter needed to spin up a new node, it would use the latest AMI in the AL2 family.
+* With that `amiFamily` set, any time Karpenter spins up a new node, it uses the latest [Amazon EKS optimized Amazon Linux 2 AMIs](https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-ami.html) release.
 * Later, if an existing node needs to be replaced, Karpenter checks to see if a newer AMI in the AL2 family is available and automatically uses the new AMI instead to spin up the new node. In other words, you may automatically get an AMI that you have not tested with your workloads.
 
 You can manually delete a node managed by Karpenter, which will cause the default behavior just described to take effect.
 However, there are situations that will cause node replacements with newer AMIs to happen automatically.
 These include: 
 
-* **Expiration**: If node expiry is set for a node, the node is marked for deletion after a certain time.
+* **Expiration**: If node expiry is set for a node, the node is marked for deletion at a certain time after the node is created.
 * [**Consolidation**]({{< relref "../concepts/disruption/#consolidation" >}}): If a node is empty of workloads, or deemed to be inefficiently running workloads, nodes can be deleted and more appropriately featured nodes are brought up to consolidate workloads.
-* [**Drift**]({{< relref "../concepts/disruption/#drift" >}}): Nodes are set for deletion when they drift from the desired state of the `NodeClaim`s and new nodes are brought up to replace them.
+* [**Drift**]({{< relref "../concepts/disruption/#drift" >}}): Nodes are set for deletion when they drift from the desired state of the NodeClaims and new nodes are brought up to replace them.
 * [**Interruption**]({{< relref "../concepts/disruption/#interruption" >}}): Nodes are sometimes involuntarily disrupted by things like Spot interruption, health changes, and instance events, requiring new nodes to be deployed.
```

Comment:
What was your thought on just dropping this in favor of L33? It's essentially duplicated here. Maybe if you wanted to call out the methods you could merge it with L33 as just a comma delimited list.

Reply:
Okay, I'll put it on a single line.